Yosemite/YosemiteTests/Stores/OrderStoreTests.swift

```diff
@@ -563,6 +563,114 @@ final class OrderStoreTests: XCTestCase {
         wait(for: [expectation], timeout: Constants.expectationTimeout)
     }
 
+    func test_optimistic_update_order_customer_note_correctly() {
+        let expectation = self.expectation(description: "Update Order Customer note")
+        let orderStore = OrderStore(dispatcher: dispatcher, storageManager: storageManager, network: network)
+
+        /// Insert Order [Customer note == "Updated!"]
+        orderStore.upsertStoredOrder(readOnlyOrder: sampleOrderMutated2(), in: viewStorage)
```

Comment:
At first, this was my approach but I changed to be consistent with the current test methods of this file, but yes, I agree with you that it make it easier to follow what is mutated. Let me change it 👍🏼

Reply:
Please feel free to diverge from how the code was previously written - some parts of it can be outdated and don't smell good so there's always space for improvements!

Reply:
Sure! thanks 🙂